lib/deck.ex

```diff
@@ -151,6 +151,18 @@ defmodule Deck do
     deck |> Enum.drop(n_cards)
   end
 
+  @doc """
+  "Burns" specified cards in the `deck` and returns the rest of the `deck`.
+
+  ## Example
+
+      iex> Deck.burn(Deck.new(8), ["Ah", "As", "Kd"])
+      ["Kc", "Kh", "Ks", "Ac", "Ad"]
+  """
+  def burn(deck, cards) when is_list(deck) and is_list(cards) do
```

Comment:
Thanks for the PR!

Usually `burn` in poker means you just discard N cards and this method already exists.

A better naming for what you need would be something like `Deck.except(deck, cards)`. This kind of naming I can accept.